src/core.js

```diff
@@ -868,6 +869,7 @@ $.extend( $.validator, {
 		startRequest: function( element ) {
 			if ( !this.pending[ element.name ] ) {
 				this.pendingRequest++;
+				$(element).addClass(this.settings.pendingClass);
```

Comment:
Please add one space before `(` and `)`

See http://contribute.jquery.org/style-guide/js/#spacing